core/conf/reader_test.go

```diff
@@ -0,0 +1,67 @@
+package conf
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestJSONReader(t *testing.T) {
+
+	reader, _ := NewReaderFromJSON(`{
+		"chain_id":"0afc093ffb509f059c55478bc1a60351cef7b4e9c008a53a6cc8241ca8617dfe",
```

Comment:
Could you please make values more dummy and not keep actual in the repo, even it's dev chain but it might give security issues